Bottom: ffceb3804a36efe7cba8b4f4ad59bf8ae403f5f1
Top:    aa171eb759345d052defef82286c037cb0823886
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-24 02:25:41 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
